src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -309,6 +309,15 @@ public void setStatusForAjax(String statusForAjax) {
         this.statusForAjax = statusForAjax;
     }
 
+    /**
+     * Retrieves the link to submit the request for copy of session. 
+     * Also contains feedback page link to return after the action.
+     * @return form submit action link
+     */
+    public String getEditCopyActionLink() {
+        return editCopyActionLink;
```

Comment:
yes, I remember that the version of JSTL prevents us from calling methods with parameters. I was thinking that we can make this method (getEditCopyActionLink) wrap the <code>getInstructorFeedbackEditCopyActionLink</code> instead.